### DISCUSSION TITLE
Place random trees

### DISCUSSION THREAD

**Discussion by Krutonium** [OpenRCT2/OpenRCT2#15645] [Feature requests]
I feel like playing trees individually is such a hassle... I think (If possible) it would be cool to be able to lay down trees by dragging your mouse around an area of land. It would also be cool to pick the types of trees to lay down in a mix so if you don't want palm trees and other undesirable trees with your forestry type trees you don't have to. Thx for reading an I hope you understand my idea. :D Pls leave comments so I can see your thoughts on my idea. :D

https://openrct.net/forum/topic/64/last/easier-tree-placing

**Comment by vzybilly** [comment]
Or a better way, make new scenery objects from groups of scenery and also being able to drag it to paint it across?

I could see the making new work nicely for building a house or something with scenery then being able to save it for use in other parks as well, maybe saving it as a new custom object or having a file in OpenRCT2 root or something that holds these custom made ones... I really see this has a nice enhancement

**Comment by rctdude2** [comment]
Perhaps it could work like in scenario editor where you can select the one button to place trees randomly in a small area. Could also probably implement the other button available in Locomotion that selects random tree types to place in the area.

**Comment by sjoerddebruin** [comment]
@rct2dude That first button is already there. The other button sounds interesting.

**Comment by IntelOrca** [comment]
Sounds like the tree place tool in OpenTTD.

**Comment by Gymnasiast** [comment]
The interface feature might be better suited for a plug-in, but identifying what is a tree or not is currently in development (object PR: https://github.com/OpenRCT2/objects/pull/22 )

**Comment by Broxzier** [comment]
Dragging could be implemented without adding a new button, and selecting multiple objects as well (Ctrl+click, or simply toggling when clustering is enabled). This will be useful for non-tree objects too.

**Comment by J-PIE-314** [comment]
PR #10357 added a feature that may have satisfied your request.

**Reply by sjoerddebruin** [reply]
That still needs you to select trees that look good with each other. This feature request is about a bunch of tree designs that get randomly placed.

**Reply by MASHzj** [reply]
With the scenery manager plugin you can select up to five different scenery items, set percentages for each and choose the area size to scatter them.

**Reply by sjoerddebruin** [reply]
That sounds fine for a user who has a sense of style and creativity. An easier version with pre-selected trees sounds like perfect for the average user.

**Reply by MASHzj** [reply]
Ok, but such a feature seems much better for a plugin than in the default game.
